doc/source/users.rst

```diff
@@ -9,5 +9,6 @@ User Guide
    utilities
    tricks
    faq
+   migrate0_1
```

Comment:
Should _migrations/_ be a subfolder?

Reply:
Hopefully there will  be few of these necessary, but  given that these should be permalinks let's do it.